scripts/eth_group_uio.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
```

Comment:
This file is a duplicate of the opaeuio.py script, which is part of opae.admin.  I think it can be safely removed.

Reply:
remove eth_group_uio.sh ,  use opaeuio.py script

opaeuio dfl_dev.10 -i -u lab -g lab 
opaeuio dfl_dev.11 -i -u lab -g lab

Reply:
fixed